ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckTokenValidator.java

```diff
@@ -86,14 +67,9 @@ public SpringSecCheckTokenValidator(SpringSecCheckAuthModule.Config config, Http
 		);
 	}
 
-	private String buildBasicAuthHeader(String user, String password) {
-		String encodedCreds = Base64.getEncoder().encodeToString((user + ":" + password).getBytes());
-		return "Basic " + encodedCreds;
-	}
-
 	private static ObjectMapper buildObjectMapper() {
 		ObjectMapper objectMapper = new ObjectMapper();
-		objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+		objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
```

Comment:
CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES was deprecated by Jackson in favor of SNAKE_CASE